sdk/cosmos/sdk/src/main/java/com/azure/data/cosmos/sync/CosmosSyncItemResponse.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.data.cosmos.sync;
+
+import com.azure.data.cosmos.CosmosItemProperties;
+import com.azure.data.cosmos.CosmosItemResponse;
+import com.azure.data.cosmos.PartitionKey;
+
+public class CosmosSyncItemResponse {
+    private CosmosItemResponse responseWrapper;
+    private CosmosSyncItem item;
+
```

Comment:
Final variables, here and other places too, wherever possible.

Reply:
Done